ansible_collections/arista/avd/examples/dual-dc-l3ls/ansible.cfg

```diff
@@ -0,0 +1,4 @@
+[defaults]
+inventory=inventory.yml
+jinja2_extensions = jinja2.ext.loopcontrols,jinja2.ext.do,jinja2.ext.i18n
+collections_paths = ../../../../../../ansible-avd:../../../../../../ansible-cvp
```

Comment:
Are the collection paths required in this example? I'm curious if the user has AVD installed in default locations when following these examples vs having the entire repository